rhp/v4/rhp_test.go

```diff
@@ -19,3 +20,212 @@ func TestMinRenterAllowance(t *testing.T) {
 		t.Fatalf("expected %v, got %v", expected, minAllowance)
 	}
 }
+
+func TestRenewalCost(t *testing.T) {
+	const (
+		initialProofHeight = 1000
+		initialExpiration  = initialProofHeight + ProofWindow
+
+		renewalHeight      = 150
+		extension          = 10
+		renewalProofHeight = initialProofHeight + extension
+		renewalExpiration  = renewalProofHeight + ProofWindow
+		renewalDuration    = renewalExpiration - renewalHeight
+	)
+	cs := consensus.State{}
+	prices := HostPrices{
+		ContractPrice:   types.NewCurrency64(100),
```

Comment:
The tip here is implicitly set to 0. Can you set it to something > 0?

Reply:
It's changed below, so I'm not sure it matters if the initial formation height is 0.